src/components/course/CourseSidebar.jsx

```diff
@@ -34,14 +35,14 @@ export default function CourseSidebar() {
   return (
     <>
       <ul className="pl-0 mb-5 course-details-sidebar">
-        {activeCourseRun.weeksToComplete && (
+        {isDefined(activeCourseRun.weeksToComplete) && (
```

Comment:
weeksToComplete and minEffort can both be 0, which if evaluating as ```if (minEffort) {}``` where minEffort is 0 you won't hit that if block. Additionally ```activeCourseRun.minEffort) && isDefined(activeCourseRun.maxEffort``` will actually display an integer in the sidebar

Reply:
🔥 good catch! i noticed this yesterday on stage and forgot to file a ticket for it haha. don't need to now :D